### DISCUSSION TITLE
Problem with the projection with _imageCoordinateSystem in the BitmapLayer

### DISCUSSION THREAD

**Discussion by NadineSarah** [visgl/deck.gl#6213] [Q&A] [answered]
Hi, 
I'm currently working with the BitMapLayer. I need to project my image from longitude/latitude to WebMercator. Therefore I'm using the _imageCoordinateSystem property. But it doesn't work, especially in the pole regions. 
As background information: I'm using mapbox js as basemap and I'm trying to visualize temperature data from January 2021 (from this website https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels-monthly-means?tab=form). The image is currently created with python (mathplotlib imshow).  In the end I want to adjust the bitmap layer class, so that it takes the temperature data and then displays the projected data as an colormap image. Therefore, I need to project the data in the bitmaplayer class.  

This is how it looks without the projection: 
<img width="594" alt="tempMapWithOutProjection" src="https://user-images.githubusercontent.com/39406788/133990571-f0d6b9d8-64f6-4554-947a-310a965d878d.PNG">

And here with the projection
<img width="600" alt="tempMapWithProjection" src="https://user-images.githubusercontent.com/39406788/133989173-b43af06b-a479-4366-9fa6-ee0ebcfd3252.PNG">

code: 

```javascript
const newLayer = new BitmapLayer({
        id: 'bitmap',
        bounds: [-180, -85.0511287798066, 180, 85.0511287798066],
        image: data,
        opacity: 0.7,
        _coordinateSystem: COORDINATE_SYSTEM.LNGLAT, 
    });
```

In the end it should look like this (this is with the projection with python and NOT with the deck gl property):
<img width="600" alt="tempMapWithPythonProjection" src="https://user-images.githubusercontent.com/39406788/133992286-e6497df2-decc-4a8a-a029-5c72a6f0451c.PNG">

Has anyone an idea what the problem might be?

**Comment by Pessimistress** [comment, marked as answer]
It looks like your image is uniformly compressed on the y axis. Are you sure the image's latitude range matches that specified in `bounds`?

**Reply by NadineSarah** [reply]
Oh - you are right the ranges are not matching! Thanks a lot for your help!
